src/Cosmos/ProtobufSerialization.cpp

```diff
@@ -13,11 +14,13 @@
 #include "Protobuf/tx.pb.h"
 #include "Protobuf/crypto_secp256k1_keys.pb.h"
 #include "Protobuf/ibc_applications_transfer_tx.pb.h"
+#include "Protobuf/terra_wasm_v1beta1_tx.pb.h"
```

Comment:
```suggestion
#include "Protobuf/cosmos_wasm_v1beta1_tx.pb.h"
```

should we use as cosmos?

Reply:
Terra uses a was fork with slightly different naming, hence it's kept under terra name.  See also comment below.